pages/api/submitForm.ts

```diff
@@ -0,0 +1,121 @@
+import { type NextApiRequest, type NextApiResponse } from 'next';
+import { contactSubject } from '../../config';
+import z from 'zod';
+
+// Zod Schema
+interface FormData {
+  Name: string;
+  Subject: string;
+  Email: string;
+  Message: string;
+}
+const formDataSchema = z.strictObject({
+  Name: z.string().max(200),
```

Comment:
Whilst technically a name could be as big as possible, I think we should simplify it to maybe 100 characters. I think that leaves room for stuff such as: `name lastname, generic business co ltd incorporated, subsidiary of other business co ltd` which i think would be the upper echelon of names